scripts/dockerfiles/readiness-probe.sh

```diff
@@ -23,9 +23,10 @@ update_readiness() {
     local svcacct=/var/run/secrets/kubernetes.io/serviceaccount
     curl --silent \
         --cacert "${svcacct}/ca.crt" \
-        -H "Authorization: bearer $(cat "${svcacct}/token")" \
-        -H 'Content-Type: application/merge-patch+json' \
-        -X 'PATCH' \
+        --header "Authorization: bearer $(cat "${svcacct}/token")" \
+        --header 'Content-Type: application/merge-patch+json' \
+        --output /dev/null \
```

Comment:
I believe this is the cause of the random JSON we sometimes see — it's the patch output (because `--silent` doesn't remove the output).

Reply:
`curl` outputs the command verbosity to standard error and the response to standard out.
Another way of doing this is:

`curl ... 2> /dev/null` as an alternative to `--silent`.
`curl ... 1> /dev/null` as an alternative to `--output=/dev/null`.

_Just thinking out loud_

Reply:
Yep, but I wanted to be sure I'm not missing _other_ messages.